src/js/apps/patients/worklist/worklist_state.js

```diff
@@ -159,7 +159,7 @@ export default Backbone.Model.extend({
 
     this.set({
       [listName]: newList,
-      lastSelectedIndex: selectedIndex,
+      lastSelectedIndex: isSelected ? selectedIndex : null,
```

Comment:
I missed this when doing the refactor in the previous commit/s.

This clears the `state.lastSelectedIndex` value (set to `null`) when a list item is un-selected.